include/application/tasks/ventilation.h

```diff
@@ -9,11 +9,20 @@
 
 #define MAX_CIRCUIT_PRESSURE_FOR_OPENING_INS_VALVE_CSP 32 //todo replace this value by the real value
 
+#include <stdint.h>

```

Comment:
optional 

by convention the include comes before the define statements (note that the ifndef should alwasy be at the top"